Snowflake/Snowflake_Ingest_json_data_from_local_stage.ipynb

```diff
@@ -0,0 +1,641 @@
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Let's not forget about `snowflake.execute` functionality\n",
+    "snowflake.execute('SHOW FILE FORMATS;')['results']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating file format with wrong parameter (not applicable for JSON file format, see [docs](https://docs.snowflake.com/en/sql-reference/sql/create-file-format.html))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Something went wrong!\n",
+      "002135 (42601): SQL compilation error:\n",
+      "Option ESCAPE is not valid for file format type JSON.\n"
+     ]
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    snowflake.file_format.create('my_invalid_son_format', 'JSON', or_replace=True, ESCAPE = '\"\\\\\"')\n",
+    "except ProgrammingError as pe:\n",
+    "    print('Something went wrong!')\n",
+    "    print(pe)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dropping file format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# snowflake.file_format.drop('my_json_format', if_exists=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating internal stage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'results': [('Stage area MY_INTERNAL_STAGE successfully created.',)],\n",
+       " 'description': [ResultMetadata(name='status', type_code=2, display_size=None, internal_size=16777216, precision=None, scale=None, is_nullable=True)],\n",
+       " 'statement': 'CREATE OR REPLACE STAGE my_internal_stage FILE_FORMAT = my_json_format'}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "snowflake.stage.create(\n",
+    "    stage_name='my_internal_stage', \n",
+    "    or_replace=True,\n",
+    "    file_format_name='my_json_format'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Putting data on the stage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "snowflake.stage.put(\n",
+    "    filepath='file://~/Downloads/reviews_data.json',\n",
```

Comment:
`filepath` value should be `file://reviews_data.json.csv` instead of `file://~/Downloads/reviews_data.json.csv`

Reply:
The same issue as above, I left a comment there so resolving this topic